src/content/data-streams/tutorials/streams-direct/solana-offchain-report-verification.mdx

```diff
@@ -0,0 +1,335 @@
+
+### Prerequisites
+
+Before you begin, you should have:
+
+- Familiarity with [Rust](https://www.rust-lang.org/learn) programming
+- Understanding of [Solana](https://solana.com/docs) concepts:
+  - [RPC Clients](https://solana.com/docs/rpc) and network interaction
+  - [Transactions](https://solana.com/docs/core/transactions) and signing
+  - [Keypairs](https://solana.com/docs/intro/wallets#keypair) and account management
+- An allowlisted account in the Data Streams Access Controller ([contact us](https://chainlinkcommunity.typeform.com/datastreams?typeform-source=docs.chain.link#ref_id=docs) to get started with Early Access)
+
+### Requirements
+
+To complete this guide, you'll need:
+
+- **Rust and Cargo**: Install the latest version using [rustup](https://rustup.rs/). Run <CopyText text="rustc --version" code/> to verify your installation.
+
+- **Solana CLI tools**: Install the latest version following the [official guide](https://docs.solana.com/cli/install-solana-cli-tools). Run <CopyText text="solana --version" code/> to verify your installation.
+
+- **Anchor Framework**: Follow the [official installation guide](https://www.anchor-lang.com/docs/installation). Run <CopyText text="anchor --version" code/> to verify your installation.
+
+- **Devnet SOL**: You'll need devnet SOL for transaction fees. Use the [Solana CLI](https://docs.solana.com/cli/transfer-tokens#airdrop-some-tokens-to-get-started) or the [Solana Faucet](https://faucet.solana.com/) to get devnet SOL. Check your balance with <CopyText text="solana balance" code/>.
+
+- **Allowlisted Account**: Your account must be allowlisted in the Data Streams Access Controller.
+
+> **Note**: While this guide uses the Anchor framework for project structure, you can implement offchain verification using any Rust-based Solana project setup. The verifier SDK and client libraries are written in Rust, but you can integrate them into your preferred Rust project structure.
+
+### Implementation guide
+
+#### 1. Create a new Anchor project
+
+1. Create a new Anchor project:
+
+   ```bash
+   anchor init example_verify
+   cd example_verify
+   ```
+
+1. Create a binary target for the verification tool:
+   ```bash
+   mkdir -p programs/example_verify/src/bin
+   touch programs/example_verify/src/bin/main.rs
+   ```
+
+#### 2. Configure your project's dependencies
+
+Update your program's manifest file (`programs/example_verify/Cargo.toml`):
+
+```toml
+[package]
+name = "example_verify"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+
+[[bin]]
+name = "example_verify"
+path = "src/bin/main.rs"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+chainlink_solana_data_streams = { git = "https://github.com/smartcontractkit/chainlink-solana", branch = "develop", subdir = "contracts/crates/chainlink-solana-data-streams" }
```

Comment:
this should be internally referenced by `sdk-off-chain` so it shouldn't be needed in this example